README.md

```diff
@@ -60,10 +63,17 @@ _Note_: Feel free to use the `nightly` tag to test the latest version, i.e. `git
 ```bash
 tar -xzf openvscode-server-v${OPENVSCODE_SERVER_VERSION}.tar.gz
 cd openvscode-server-v${OPENVSCODE_SERVER_VERSION}
-./server.sh
+./bin/openvscode-server # you can add arguments here, use --help to list all of the possible options
 ```
+
 - Visit the URL printed in your terminal. Just going to port 3000 won't work, because VS Code requires that you provide a uniquely generated security token to prevent unauthorized access.
 
+#### A note about security tokens
+
+Since OpenVSCode Server v1.62, you can access the Web UI using a connection token generated by the server. If that's not what you're looking for, and instead want to generate these tokens by yourself, start the server with `--connection-token YOUR_TOKEN` to force your secret to be used. If you want your development environment to be even more secure, create a plaintext file with the desired token as its contents and provide it to the server with `--connection-secret YOUR_SECRET_FILE`. By default, a token will always be generated.
```

Comment:
State here that since v1.64 we use `--without-connection-token` by default, if user wants to use a token then can modify the dockerfile or use docker compose `entrypoint` option